### ISSUE TITLE
Feature for make osx El Capitan mouse larger temporarily

### ISSUE THREAD

**Issue by Jacob201311** [kasper/phoenix#182]
> To make the pointer larger temporarily, quickly move your finger on the trackpad or quickly move the mouse.
https://support.apple.com/kb/PH21543?locale=en_US

Sometimes when I mv mouse to a position, make the mouse temporarily larger would be useful. 
So, Mouse need a function like Mouse.showUp()

**Comment by Jacob201311** [comment]
I think I should mock mouse shake first

**Comment by kasper** [comment]
Yes, I’m not sure if there’s an API to achieve this directly. Of course you can just mock the shake feature by rapidly changing/shaking the position of the pointer and see if that causes the effect.

**Comment by Jacob201311** [comment]
I mock the mouse to shake, but it doesn't work, may be I did not figure out the trigger condition。
Instead, I use modal to solve my problem.

**Comment by mafredri** [comment]
> I mock the mouse to shake, but it doesn't work, may be I did not figure out the trigger condition

It seems `Mouse.move` simply moves the mouse pointer, but does not seem to *trigger* a mouse move in macOS. For example, if I'm currently dragging an item and use Phoenix to move the pointer somewhere else, the dragged item will not move until I move the physical mouse around.

**Comment by kasper** [comment]
Yes, @mafredri is likely right on point.

**Comment by kasper** [comment]
Seems like there is a private API to change the cursor size: `CGSGetCursorScale` and `CGSSetCursorScale`. 🙈
